lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -416,68 +428,108 @@ spec = do
+    describe "STAKE_POOLS_LIST_01 - List stake pools" $ do
+            it "immediately has non-zero saturation & stake" $ \ctx -> do
+                r <- listPools ctx
+                expectResponseCode HTTP.status200 r
+                verify r
+                    [ expectListSize 3
+                    -- At the time of setup, the pools have 1/3 stake each, but
+                    -- this could potentially be changed by other tests. Hence,
+                    -- we try to be forgiving here.
+                    , expectListField 0
+                        (#metrics . #relativeStake)
+                            (.> Quantity (unsafeMkPercentage 0))
+                    , expectListField 1
+                        (#metrics . #relativeStake)
+                            (.> Quantity (unsafeMkPercentage 0))
+                    , expectListField 2
+                        (#metrics . #relativeStake)
+                            (.> Quantity (unsafeMkPercentage 0))
+                    , expectListField 0
+                        (#metrics . #saturation) (.> 0)
+                    , expectListField 1
+                        (#metrics . #saturation) (.> 0)
+                    , expectListField 2
+                        (#metrics . #saturation) (.> 0)
+                    ]
+            it "eventually has correct margin and cost" $ \ctx -> do
+                eventually "pool worker finds the certificate" $ do
+                    r <- listPools ctx
+                    expectResponseCode HTTP.status200 r
+                    verify r
+                        [ expectListField 0
+                            (#cost) (`shouldBe` Just (Quantity 0))
+                        , expectListField 1
+                            (#cost) (`shouldBe` Just (Quantity 0))
+                        , expectListField 2
+                            (#cost) (`shouldBe` Just (Quantity 0))
+
+                        , expectListField 0
+                            #margin (`shouldBe` Just
+                                (Quantity $ unsafeMkPercentage 0.1 ))
+                        , expectListField 1
+                            #margin (`shouldBe` Just
+                                (Quantity $ unsafeMkPercentage 0.1 ))
+                        , expectListField 2
+                            #margin (`shouldBe` Just
+                                (Quantity $ unsafeMkPercentage 0.1 ))
+
+                        -- TODO: Switch from (.>=) to (.>) (and perhaps an eventually) when
+                        -- our stake pool cluster are producing blocks.
+                        , expectListField 0
+                            (#metrics . #producedBlocks) (.>= Quantity 0)
+                        , expectListField 1
+                            (#metrics . #producedBlocks) (.>= Quantity 0)
+                        , expectListField 2
+                            (#metrics . #producedBlocks) (.>= Quantity 0)
```

Comment:
Now after rebasing I see them producing blocks, but I still see no rewards 🤔

Reply:
> I think some monetary protocol parameters need to be tweaked in the genesis file

That is possible.

Reply:
I created the related https://github.com/input-output-hk/cardano-wallet/pull/1783, which might be good for investigating. We can leave the todo for rewards in this PR for now.